index.html

```diff
@@ -53,6 +53,218 @@ <h1>I'm Mariane <br/>
                 </div>
             </div>
         </section>
+        <section id="card-section">
+            <div class="card-container">
+                <div class="cards">
+                    <div>
+                        <img src="asset/snapshot1.svg" alt="screenshot" class="screenshot">
+                    </div>
+                    <div class="card-description">
+                        <div>
+                            <p class="card-title">Tonic</p>
```

Comment:
- [ ] Kindly make sure to wrap the title of the project into the heading `hx` tag. :wink: